Checks/PWR003/example.f90

```diff
@@ -1,23 +1,26 @@
 ! PWR003: Explicitly declare pure functions
 
-module example_module
-   implicit none
-   integer :: c = 1
-contains
-   ! Depends on external data (c)
-   ! No side effects
-   pure function example_pure(a) result(res)
-      integer, intent(in) :: a
-      integer :: res
-      res = a + c
-   end function example_pure
+! Depends only on its arguments
+! No side effects
+function gravity()  
+  real :: gravity
+  gravity = 9.8 ! m/s²
+  return
+end function gravity
```

Comment:
```suggestion
! Returns the gravity on Earth in m/s2
function gravity()
  use iso_c_binding, only : c_double
  implicit none
  real(kind=c_double) :: gravity
  gravity = 9.8
end function gravity
```

Use `implicit none` and `c_double`. Also, no need to call `return`.

Reply:
OMW!